components/content_settings/core/browser/brave_content_settings_default_provider.cc

```diff
@@ -26,9 +26,6 @@ BraveDefaultProvider::~BraveDefaultProvider() {}
 std::unique_ptr<RuleIterator> BraveDefaultProvider::GetRuleIterator(
     ContentSettingsType content_type,
     bool off_the_record) const {
-  if (content_settings::IsShieldsContentSettingsType(content_type))
```

Comment:
why this change is needed?

Reply:
i think i've got the idea, but if we remove this check then the whole class seems useless

Reply:
indeed, I forgot to remove this once I verified that everything was working correctly